Objects/tupleobject.c

```diff
@@ -59,6 +59,15 @@ show_track(void)
 }
 #endif
 
+static inline void
+TUPLE_GC_TRACK(PyTupleObject *op)
```

Comment:
nitpick: hum, since it's no longer a macro, I suggest to rename it to tuple_gc_track().